### PR TITLE
feat: add KeyCDN support

### PR DESCRIPTION
**Pull request by tiptenbrink**
Hi! This should add support for KeyCDN to unpic. Let me know if anything's missing from the tests / code.

### CODE REVIEW

File: src/parse.ts [hunk 1]
```diff
@@ -12,6 +12,7 @@ import { parse as kontentai } from "./transformers/kontentai.ts";
 import { parse as vercel } from "./transformers/vercel.ts";
 import { parse as nextjs } from "./transformers/nextjs.ts";
 import { parse as scene7 } from "./transformers/scene7.ts";
+import { parse as keycdn } from "./transformers/scene7.ts";
```
Comment (ascorbic) [inline src/parse.ts:15]:
```suggestion
import { parse as keycdn } from "./transformers/keycdn.ts";
```

**Comment by ascorbic** [comment]
Great. Just one copy+paste error. Also, could you add an example image to [example.json](https://github.com/ascorbic/unpic/blob/main/demo/src/examples.json)? Thanks!

**Comment by tiptenbrink** [comment]
> Great. Just one copy+paste error. Also, could you add a sample image to the end of the list in [examples.json](https://github.com/ascorbic/unpic/blob/main/demo/src/examples.json)? Thanks!

@ascorbic Ah apologies about that! I think everything should be good now. One question though, should it always center the crop if the width/height don't match the image aspect ratio? By default it doesn't do that on KeyCDN but I could change it so that would be the default behavior.

**Comment by ascorbic** [comment]
That's great, thanks! The current cropping approach looks great. The only improvement would be, if it supports it, prevent up-scaling images. Generally the approach we want for the default behaviour is:

- maintain aspect ratio (this is required)
- use "cover" strategy (i.e. the image is cropped to fit the aspect ratio, rather than padded. also required)
- use the CDN's "best" focus strategy. Some have smart cropping. The current behaviour looks good.
- no upscaling. browsers are better at it

**Pull request merged**